stats/requested_books.php

```diff
@@ -72,10 +73,10 @@
 echo "<br>\n";
 echo "<h2>" . _("Most Requested Books Posted to Project Gutenberg") . "</h2>\n";
 
-$pg_url1 = DPDatabase::escape("<a href='http://www.gutenberg.org/ebooks/");
+$pg_url1 = DPDatabase::escape(sprintf("<a href='%s", get_pg_catalog_url_for_etext('')));
```

Comment:
do we need a migration on the table since existing entries will not have https?

Reply:
This string isn't stored in the DB so there's nothing that needs to be migrated.